tests/system/action/meeting_user/test_create.py

```diff
@@ -0,0 +1,22 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class MeetingUserCreate(BaseActionTestCase):
+    def test_create(self) -> None:
```

Comment:
Needs discussion: 
Write one more test to change the about_me-field without any right, but being member of the meeting as replacement for the field in user.update_self. You also have to change the permission check.

Reply:
I think we need a new action, like update_self, for this. This should be discussed. Removing the permission is not the solution.